### ISSUE TITLE
plot-vcfstats error

### ISSUE THREAD

**Issue by ycl6** [samtools/bcftools#355]
Hi,

I am getting below error when using plot-vcfstats. I attached the [plot-vcfstats.log](https://github.com/samtools/bcftools/files/39831/plot-vcfstats.txt) for your reference.

Thanks.

I-Hsuan

```
$ plot-vcfstats bcftools_stats.vchk -p bcftools_stats/
Parsing bcftools stats output: bcftools_stats.vchk
Plotting graphs: python bcftools_stats/plot.py
Creating PDF: pdflatex summary.tex >plot-vcfstats.log 2>&1
The command exited with non-zero status, please consult the output of pdflatex: bcftools_stats/plot-vcfstats.log

 at plot-vcfstats line 78
        main::error('The command exited with non-zero status, please consult the o...') called at plot-vcfstats line 2082
        main::create_pdf('HASH(0x13036960)') called at plot-vcfstats line 69
```

bcftools Version: 1.2 (using htslib 1.2.1)

**Comment by envanloo** [comment]
I just got the same error. First, I looked into the log file plot-vcfstat.log and it only said: 

sh: 1: pdflatex: not found

Then I looked in the python script in the target plots directory and there I found no reference to a command to carry out pdflatex. 

Next, Iooked into summary.tex as that is apparently the file that is processed. Here, in the header it says: 

% This file was produced by plot-vcfstats, the command line was:
%   plot-vcfstats -p /media/scratchpad_01/loo006/Quinoa/test_files/mp_bsa_all_region1/plots/ /media/scratchpad_01/loo006/Quinoa/test_files/mp_bsa_all_region1/quinoa_bsa_all.vcf.gz.stats
%
% Edit as necessary and recreate the PDF by running
%   pdflatex summary.tex
%

So, it looks like that 'pdflatex summary.tex'  can also be run after the plot-vcfstats has run. 

This proved to be the case running:

$pdflatex summary.tex on the Linux prompts in the target directory plots just gives you the full summary.pdf that was missing. 

Still no idea why this error occurred. Seems a bug.

**Comment by jrandall** [comment]
`plot-vcfstats` should check for the presence and basic function of `pdflatex` before attempting to compile the report, so that it can report a sensible error message in case LaTeX is not functioning properly.

**Comment by winni2k** [comment]
Uh, I think this has been fixed by now? I got an informative error to install `pdflatex` or `tectonic` when running `plot-vcfstats` from bcftools `1.10.2`.

Perhaps this issue can be closed?

**Comment by pd3** [comment]
That's correct, this was addressed by 9e2597f0. Thank you
